src/select.coffee

```diff
@@ -400,6 +400,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
     )
   .property 'groupedContent.[]'
 
+  # whether or not to ensure that highlighted option is visible
```

Comment:
I think we should rename this option to something more aligned with that description. Maybe something like `scrollSelectionIntoView`?